inc/Engine/Optimization/AssetsLocalCache.php

```diff
@@ -45,16 +45,18 @@ public function __construct( $cache_path, $filesystem ) {
 	 * @since 3.1
 	 *
 	 * @param string $url URL to get the content from.
+	 * @param bool   $save Save the resource locally or just get contents.
+	 *
 	 * @return string
 	 */
-	public function get_content( $url ) {
+	public function get_content( $url, $save = true ) {
```

Comment:
Instead of using a new parameter, how about splitting the method in 2:
- Keep this method as is
- Create a new method `get_raw_content( $url )`, which would always fetch the remote URL instead of checking if we have a local cache first
- Then use this new method inside `get_content()` for when the content needs to be fetched

That way we avoid a boolean parameter, which is usually discouraged (breaks single responsability), and split the complexity into 2 methods instead of one.